scripts/lua/header_authenticated.lp

```diff
@@ -68,7 +68,7 @@ mg.include('header.lp','r')
                         <ul class="dropdown-menu">
                             <!-- User image -->
                             <li class="user-header">
-                                <img class="logo-img" src="<?=pihole.webhome()?>img/logo.svg" alt="Pi-hole Logo" style="border: 0" width="90" height="90">
+                                <img class="logo-img" src="<?=pihole.webhome()?>img/logo.svg" alt="Pi-hole Logo" width="50" height="50">
```

Comment:
Why did you change the dimensions?

Reply:
Because that was the final image size set in the CSS:

https://github.com/pi-hole/web/blob/25441178f7dcc365c5a553a86b23eeed0573938f/style/pi-hole.css#L929-L933